clusterctl/cmd/validate.go

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var validateCmd = &cobra.Command{
+	Use:   "validate",
+	Short: "Validate an API resource created by cluster API.",
+	Long:  "Validate an API resource created by cluster API.",
```

Comment:
I would actually suggest more details on the validate cluster command instead of the validate command as I suspect validation will be cluster API resource-specific.

Reply:
I agree with Jessica.

And  I think it is very early and there is no real implementation yet. I intend to update the help message as I add more stuff in follow-up PRs.